app/models/ingredient.py

```diff
@@ -4,5 +4,10 @@
 
 class Ingredient(BaseModel):
     id: UUID4
+    amount: float  # maybe make a way to scale the recipe by multiplying the amount variable?
+    measurementType: str  # it should be possible to leave this blank, i.e. 1 ___ lemon vs 1 tsp cumin
+    # ^^^^^ maybe a shorter variable name?

```

Comment:
That's probably fine in terms of length. You could turn the str into an enum, potentially, though.